### PR TITLE
DOCS-7206 added troubleshooting nav

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 1]
```diff
@@ -0,0 +1,125 @@
+= Binding Your Service to an API Using the CLI
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:1]: Instructions for review:
Now I need you to look at new changes and do a sanity check for all content.

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, binding
+
+When you create the binding, Anypoint Service Mesh creates a three-way association of the service in Kubernetes, the API that you created in Anypoint Platform to represent the service, and the adapter that you created for the service. The adapter uses the unique API ID to fetch policies and apply them to your services that are running in the cluster.
+
+If you created the API in the previous procedure using Anypoint Platform, you use the API ID to bind the service. However, if you you used Anypoint Service Mesh to create the API, you must use the API name to bind the service.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:9]: Cannot refer to "previous"; need to name the text section or omit:
```suggestion
If you created the API using Anypoint Platform, you use the API ID to bind the service. However, if you you used Anypoint Service Mesh to create the API, you must use the API name to bind the service.
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+NAMESPACE      NAME        ADAPTER                         API ID  SERVICE NAME    STATUS
+nto-payments   customers   anypoint-service-mesh-adapter   1234    customers       Ready
+----
+
+== Methods for Binding an API
+
+You can bind an API to a service using Anypoint Platform or Anypoint Service Mesh, based on your preference:
+
+* <<bind_the_service, Bind an API created using Anypoint Platform>>: If you created your API using Anypoint Platform, use this procedure.
+* <<bind_using_ASM, Bind an API created using Anypoint Service Mesh>> : If you created your API using Anypoint Service Mesh, use this procedure.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:31]: remove space
```suggestion
* <<bind_using_ASM, Bind an API created using Anypoint Service Mesh>>: If you created your API using Anypoint Service Mesh, use this procedure.
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+[[bind_the_service]]
+== Bind an API Created Using Anypoint Platform
+
+Before you bind the service, ensure that you complete the task prerequisite.
+
+*Task Prerequisite*
+From the API Administration Tab of API Manager, obtain the API ID to create the binding using this method.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:39]: Before you bind the service, from the API Administration Tab of API Manager, obtain the API ID.

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+To bind the service with an API, perform the following steps:
+
+. Go to *Anypoint Platform > API administration* and expand the API version name to display the instances.
+. Click an instance name and copy the API ID that is displayed. 
+Ensure that you save this API ID to a text editor.
++
+image::api-id.png[]
++
+. To retrieve the service name from Kubernetes, with which you want to bind the API, run the command:
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:49]:
```suggestion
. Retrieve the service name from Kubernetes, with which you want to bind the API:
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
++
+`kubectl get svc -n <namespace>`
+. Verify that your microservice port is named according to the requirements described in xref:http://istio.io/docs/setup/additional-setup/requirements/[Istio].
+. From the Anypoint Service Mesh installer download location in the Command window, type the following command to bind the adapter with an application:
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:53]:
```suggestion
. From the Anypoint Service Mesh installer download location in the Command window, bind the adapter with an application:
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+--namespace=<namespace name> \
+--adapter=<adapter name> \
+--apiId=<api ID> \
+--serviceName=<service name>
+----
+Note that all object names, such as adapter name, binding name, and service name, must be in lowercase letters. An error is thrown if you do not follow Kubernetes naming conventions.
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:68]:
```suggestion
| Parameter Name | Description | Required or Optional | Default Value
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+| name | The unique name that represents the binding of the Kubernetes Service with the API, for example, `service-binding-anypoint-security`. 
+ | Required | Not applicable
+| namespace | The namespace in which your service is deployed. Ensure that you use the same namespace as the adapter you provisioned in the previous procedure. | Required | Not applicable
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:71]:
```suggestion
| namespace | The namespace in which your service is deployed. Ensure that you use the same namespace as the adapter you provisioned earlier. | Required | Not applicable
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+| adapter | Specify the name of the adapter that you provisioned in the previous procedure.  | Required | Not applicable
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:72]:
```suggestion
| adapter | Specify the name of the adapter that you provisioned earlier.  | Required | Not applicable
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+| apiId | Specify the API ID of the API that you want to associate with this adapter. Use the API ID you obtained as part of Review Prerequisites section of this document. | Required | Not applicable
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:73]:
```suggestion
| apiId | Specify the API ID that you acquired prior to beginning this procedure. | Required | Not applicable
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+| serviceName | The name of the service that you want to bind to the API, which will be managed by Anypoint Platform. The service name is the identifier of the service that you want to manage with Anypoint Service Mesh. | Required | Not applicable
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:74]: Is the service managed by Anypoint Platform or Anypoint Service Mesh? The two sentences sound contradictory.

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+|===
++
+The binding is activated within a few seconds and the API is successfully registered to the proxy.
+. Verify the binding status using the following command:
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:78]:
```suggestion
. Verify the binding status:
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
++
+After you bind your API, the API in API Manager is marked as Active. You are now ready to apply policies to protect your APIs:
++
+image::api-manager-status-for-service-mesh.png[85%, 85%]
+
+[[bind_using_ASM]]
+
+== Bind an API Created Using Anypoint Service Mesh
+
+Before you bind the service, ensure that you complete the task prerequisite.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:90]: Copy the text suggested above, at the beginning of the previous task.

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 14]
```diff
@@ -0,0 +1,125 @@
...
+
+To create the binding:
+
+. Retrieve the service name from Kubernetes, with which you want to bind the API using the following command:
+`kubectl get svc -n <namespace>`.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:95]:
```suggestion
`kubectl get svc -n <namespace>`
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 15]
```diff
@@ -0,0 +1,125 @@
...
+. Verify that your microservice port is named based on xref:https://istio.io/docs/setup/additional-setup/requirements/[Istio requirements].
+. From the Anypoint Service Mesh installer download location, type the following command to bind the adapter with an API:
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:97]:
```suggestion
. From the Anypoint Service Mesh installer download location, bind the adapter with an API:
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 16]
```diff
@@ -0,0 +1,125 @@
...
+--name=<binding name> \
+--namespace=<namespace name> \
+--adapter=<adapter name> \
+--api=<api name> \
+--serviceName=<service name>
+----
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:111]:
```suggestion
| Parameter Name | Description | Required or Optional | Default Value
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 17]
```diff
@@ -0,0 +1,125 @@
...
+| name | The unique name that represents the binding of the Kubernetes Service with the API, for example, `service-binding-anypoint-security`.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:112]:
```suggestion
| name | The unique name that represents the binding of the Kubernetes service with the API, for example, `service-binding-anypoint-security`. 
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 18]
```diff
@@ -0,0 +1,125 @@
...
+--api=<api name> \
+--serviceName=<service name>
+----
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
+| name | The unique name that represents the binding of the Kubernetes Service with the API, for example, `service-binding-anypoint-security`. 
+ | Required | Not applicable
+| namespace | The namespace in which your service is deployed. Ensure that you use the same namespace as the adapter you provisioned in the previous procedure. | Required | Not applicable
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:114]:
```suggestion
| namespace | The namespace in which your service is deployed. Ensure that you use the same namespace as for the adapter you provisioned earlier. | Required | Not applicable
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 19]
```diff
@@ -0,0 +1,125 @@
...
+| adapter | Specify the name of the adapter that you provisioned in the previous procedure.  | Required | Not applicable
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:115]:
```suggestion
| adapter | Specify the name of the adapter that you provisioned earlier.  | Required | Not applicable 
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 20]
```diff
@@ -0,0 +1,125 @@
...
+----
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
+| name | The unique name that represents the binding of the Kubernetes Service with the API, for example, `service-binding-anypoint-security`. 
+ | Required | Not applicable
+| namespace | The namespace in which your service is deployed. Ensure that you use the same namespace as the adapter you provisioned in the previous procedure. | Required | Not applicable
+| adapter | Specify the name of the adapter that you provisioned in the previous procedure.  | Required | Not applicable 
+| api | Specify the name of the API you want to associate with this adapter. You created this API in the previous procedure. | Required | Not applicable
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:116]:
```suggestion
| api | Specify the name of the API you want to associate with this adapter.  | Required | Not applicable
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 21]
```diff
@@ -0,0 +1,125 @@
...
+| serviceName | The name of the service that you want to bind to the API, which will be managed by Anypoint Platform. The service name is the identifier of the service that you want to manage with Anypoint Service Mesh.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:117]: See previous edit to this same description above.

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 22]
```diff
@@ -0,0 +1,124 @@
+= Binding Your Service to an API Using a CRD
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, binding
+
+When you create the binding, Anypoint Service Mesh creates a three-way association of the service in Kubernetes,the API that you created in Anypoint Platform to represent the service, and the adapter that you created for the service.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:7]: add space
```suggestion
When you create the binding, Anypoint Service Mesh creates a three-way association of the service in Kubernetes, the API that you created in Anypoint Platform to represent the service, and the adapter that you created for the service. 
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 23]
```diff
@@ -0,0 +1,124 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, binding
+
+When you create the binding, Anypoint Service Mesh creates a three-way association of the service in Kubernetes,the API that you created in Anypoint Platform to represent the service, and the adapter that you created for the service. 
+
+The adapter uses the unique API ID to fetch policies and apply them to your services that are running in the cluster.
+
+If you created the API in the previous procedure using Anypoint Platform, you use the API ID to bind the service. However, if you used the CRD to create the API, you must use the API name to bind the service.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:11]:
```suggestion
If you created the API using Anypoint Platform, you use the API ID to bind the service. However, if you used the CRD to create the API, you must use the API name to bind the service.
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 24]
```diff
@@ -0,0 +1,124 @@
...
+NAMESPACE      NAME        ADAPTER                         API ID  SERVICE NAME    STATUS
+nto-payments   customers   anypoint-service-mesh-adapter   1234    customers       Ready
+----
+
+== Methods for Binding an API
+
+You can bind an API to a service using Anypoint Platform or Anypoint Service Mesh, based on your preference:
+
+* <<bind_the_service, Bind an API created using Anypoint Platform>>: If you created your API using Anypoint Platform, use this procedure.
+* <<bind_using_ASM, Bind an API created using Anypoint Service Mesh>> : If you created your API using Anypoint Service Mesh, use this procedure.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:32]: remove space
```suggestion
* <<bind_using_ASM, Bind an API created using Anypoint Service Mesh>>: If you created your API using Anypoint Service Mesh, use this procedure.
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 25]
```diff
@@ -0,0 +1,124 @@
...
+
+
+[[bind_the_service]]
+== Bind an API Created Using Anypoint Platform
+
+Before you bind the service, ensure that you complete the task prerequisite.
+
+*Task Prerequisite*
+From the API Administration Tab of API Manager, obtain the API ID to create the binding using this method.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:41]: Suggest combining as illustrated in an earlier file.

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 26]
```diff
@@ -0,0 +1,124 @@
...
+
+To bind the service with an API, perform the following steps:
+
+. Go to *Anypoint Platform > API administration* and expand the API version name to display the instances.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:45]: not nec
"to display the instances"

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 27]
```diff
@@ -0,0 +1,124 @@
...
+. Click an instance name and copy the API ID that is displayed. 
+Ensure that you save this API ID to a text editor.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:47]: Do we have to tell them where or can we say...:
```suggestion
Ensure that you save this API ID somewhere that you can access readily.
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 28]
```diff
@@ -0,0 +1,124 @@
...
++
+image::api-id.png[]
++
+. To retrieve the service name from Kubernetes, with which you want to bind the API, run the command:
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:51]:
```suggestion
. Retrieve from Kubernetes the name of the service with which you want to bind the API:
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 29]
```diff
@@ -0,0 +1,124 @@
...
++
+`kubectl get svc -n <namespace>`
+. Verify that your microservice port is named following the requirements requested by Istio: xref:https://istio.io/docs/setup/additional-setup/requirements/[].
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:54]:
```suggestion
. Verify that your microservice port is named following Istio standards: 
xref:https://istio.io/docs/setup/additional-setup/requirements/[]
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 30]
```diff
@@ -0,0 +1,124 @@
...
+. If you created your API using Anypoint Platform, use `spec.parameters.apiId`:
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:55]: "use `spec.parameters.apiId`" how?  Do you input this syntax? Where?

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 31]
```diff
@@ -0,0 +1,124 @@
...
+    service-mesh.mulesoft.com/kind: adapter-binding
+spec:
+  instanceRef:
+    name: <adapter name> # example: bookinfo-service-mesh-adapter
+  parameters:
+    apiId: <api ID> # example: 15722113
+    service: <service name> # example: orders-service
+----
++
+The binding is activated within a few seconds and the API is successfully registered to the proxy.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:74]: Do we need to say that the steps were successful by including this text?
I would omit it.

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 32]
```diff
@@ -0,0 +1,124 @@
...
+. Verify the binding status using the following command:
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:75]:
```suggestion
. Verify the binding status:
```

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc [hunk 33]
```diff
@@ -0,0 +1,124 @@
...
++
+`./service-mesh adapter binding list`
++
+After you bind your API, the API in API Manager is marked as Active. You are now ready to apply policies to protect your APIs.
```
Comment (kbostrom) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:79]:
```suggestion
The API in API Manager is marked `Active` and you can now apply policies to protect your APIs.
```

Comment (kbostrom) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:79]: Okay, I am going to stop torturing you now with ideas for minimalism. You can apply them when/as you see fit.
I'll just proof the rest.

Comment (kbostrom) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:79]: STOP HERE TO SUBMIT

**Review by kbostrom — APPROVED** [review]
DONE with the first half of DOCS-7206 added troubleshooting nav #36

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:9]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:31]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:49]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:1]: Ok great! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:39]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:53]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 7 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:68]: I thought I had searched all pages for this and fixed it. Don't know how this was missed. Thanks! Done!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:71]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:72]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 10 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:73]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 11 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:74]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 12 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:78]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 13 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:90]: This sentence not needed here anymore. Removed! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 14 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:95]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 15 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:97]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 16 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:111]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 17 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:112]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 18 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:114]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 19 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:115]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 20 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:116]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 21 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc:117]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 22 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:7]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 23 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:11]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 24 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:32]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 25 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:41]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 26 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:45]: Kept it. Users need to know why they are performing a specific task.

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 27 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:47]: Thanks! Done!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 28 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:51]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 29 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:54]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 30 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:55]: In the yaml file. Good catch. Added! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 31 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:74]: I'm keeping it as it reinforces the users and tells them what they should be seeing on their screen. Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 32 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:75]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc (hunk 33 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc:79]: Done! Thanks!

**Pull request merged**